tests/python/pytorch/graphbolt/gb_test_utils.py

```diff
@@ -165,3 +166,11 @@ def random_homo_graphbolt_graph(
                     path: {test_path}
     """
     return yaml_content
+
+
+def setup_seed(seed):
+    torch.manual_seed(seed)
+    torch.cuda.manual_seed_all(seed)
+    torch.backends.cudnn.deterministic = True
+    dgl.seed(seed)
```

Comment:
Likely we don't need set seed for dgl and dgl random, can you try remove and see whether it still passes?

Reply:
Right.